src/plugins/data_source_management/public/components/data_source_item/data_source_item.scss

```diff
@@ -0,0 +1,17 @@
+.dataSourceAggregatedViewOuiFlexItem {
```

Comment:
The newly added are specific to the component, then can we add the css to the file that belongs to the component?

Reply:
Thanks for the suggestion, already pushed the updated code